proposals/NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md

```diff
@@ -0,0 +1,68 @@
+```
+# Feature name
+
+* Proposal: [SDL-NNNN](NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md)
+* Author: [Irina Getmanets](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: Core, RPC
+
+## Introduction
+
+VRHelp items and VRHelpTitle are set by SetGlobalProperties for helping user to make voice navigation over registered application, what is very usefull in case your hands are busy with driving.
```

Comment:
@GetmanetsIrina 
- I think its good to decrypt VR first. Otherwise reader may guess is it Virtual Reality, Video Recording or something else
- VRHelp, VRHelpTitle need to be described, what do they basically define
- would be good to link SetGlobalProperties to the MOBILE_API.xml e.g. [SetGlobalProperties](https://github.com/smartdevicelink/sdl_core/blob/master/src/components/interfaces/MOBILE_API.xml#L3125)